tests/profiler/test_continuous_profiler.py

```diff
@@ -31,14 +32,29 @@ def experimental_options(mode=None, auto_start=None):
     }
 
 
+@pytest.fixture
+def sdk_info():
+    return {
+        "name": "sentry.python",  # SDK name will be overridden after integrations have been loaded with sentry_sdk.integrations.setup_integrations()
```

Comment:
Is this comment accurate here? If not, let's remove it:

```suggestion
        "name": "sentry.python",
```